bin/region_mover.rb

```diff
@@ -493,11 +493,22 @@ def getFilename(options, targetServer, port)
   return filename
 end
 
+def getServersRSGroup(rsgroup_admin, hostname, port)
+  rsgroup = rsgroup_admin.getRSGroupOfServer(Address.fromParts(hostname,
+                                             java.lang.Integer.parseInt(port)))
+  # If the rsgroup is nil, that means this server belongs to no rsgroup.
+  # It should be already offline.
+  # Here we directly log and exit.
+  return rsgroup unless rsgroup.nil?
+  $LOG.info('The server ' + hostname + 'belongs to no rsgroup. Exit regions moving.')
```

Comment:
This feels like it justifies at least a `warn` level log

Reply:
From the overall system viewpoint, this might be seen as a normal behaviour that region_mover exited gracefully as expected, however from region_mover's own viewpoint, yes it makes sense to convert it to WARN level because the same behaviour is possible only when either the given server is in zombie state or maybe some other issues persist in RSGroup APIs.